actor/message-pub/src/lib.rs

```diff
@@ -0,0 +1,40 @@
+use wasmbus_rpc::actor::prelude::*;
+use wasmcloud_interface_httpserver::{HttpRequest, HttpResponse, HttpServer, HttpServerReceiver};
+use wasmcloud_interface_messaging::{Messaging, MessagingSender, PubMessage};
+
+const MESSAGE_PREFIX: &str = "wasmcloud.http";
+
+#[derive(Debug, Default, Actor, HealthResponder)]
+#[services(Actor, HttpServer)]
+struct MessagePubActor {}
+
+#[async_trait]
+impl HttpServer for MessagePubActor {
+    async fn handle_request(
+        &self,
+        ctx: &Context,
+        req: &HttpRequest,
+    ) -> std::result::Result<HttpResponse, RpcError> {
+        // Format the path as a dot separated subject instead
+        let subject = format!("{}{}", MESSAGE_PREFIX, req.path.replace('/', "."));
```

Comment:
should we replace dots in the path (e.g. `/hello/there/bob.mcbobberson` with underscores in the NATS topic? `hello.there.bob_mcbobberson` ?

Reply:
Good idea, setting that